dts/bindings/wifi/nordic,nrf7002-qspi.yaml

```diff
@@ -15,60 +15,16 @@ properties:
     type: int
     required: true
     description: |
-      Maximum clock speed supported by the device, in Hz.
+      Maximum clock speed (in Hz) supported by the device.
 
   reg:
     required: true
 
-  readoc:
```

Comment:
I see multiple problems with this commit.

First, it's a fixup commit. For example, an earlier commit in the branch added devicetree descriptions, and this commit is changing them instead of squashing them. I explained yesterday that we do not accept fixup commits. In particular, the changes to all of these files in this commit are not correctly rebased:

- nordic,nrf7002-qspi.yaml
- nordic,nrf7002-spi.yaml
- nordic,nrf7002.yaml

Second, the changes in this file go beyond documentation changes, so this patch is doing more than one thing. There are significant semantic changes being made here to the relevant devicetree properties which were introduced previously. In particular, it looks to me like:

1. A new `quad-mode` property is **added** in this commit.
2. The following properties are **removed**  in this commit:

- `readoc`
- `writeoc`
- `address-size-32`
- `sck-delay`

What is going on here?

- If the properties which were removed are unnecessary, then why where they added in the first place?
- If the property which was added is necessary, why wasn't it added in the original commit?

The answers are not clear, and the commit logs up to this point in the pull request do not provide explanations as to why these changes are being done:

```
wifi: Add documentation for nrf7002

This adds documentation for nrf70 series, the first chip supported is
nrf7002.
```

and

```
wifi: Add nrf7002 boards support

Add nrf7002 board variants and DTS files.
```

In short, I can't tell if these changes to the bindings were made by mistake or were done on purpose, and there are clear problems with the branch structure.

This is not acceptable for merging IMO. The branch structure does not satisfy NCS requirements for a well-formed pull request. These bindings look like they still need work.

Reply:
The rebase was a mistake, so, these are in development changes and have to be merged into the original commit, will do it.